R/pkgdown.R

```diff
@@ -5,7 +5,14 @@
 #'   * `use_pkgdown()`: creates a pkgdown config file and adds the file and
 #'     destination directory to `.Rbuildignore`.
 #'   * `use_pkgdown_travis()`: helps you set up pkgdown for automatic deployment
-#'     on Travis-CI.
+#'     on Travis-CI by:
+#'       * Adding the `docs/` folder to `.gitignore` and `.Rbuildignore`.
+#'       * Optionally building favicons if your package contains a package logo,
+#'         and adding the resulting `pkgdown/` folder to `.Rbuildignore`.
+#'       * Creating an empty `gh-pages` branch for the pkgdown site to be
+#'         deployed to.
+#'       * Prompting you about what to do next regarding Travis-CI deployment
+#'         keys and updating your `.travis.yml`.
```

Comment:
Sure, I'm happy to also get better description docs. Grammar nit: I'd rewrite the lead in, so all of these bullets can be "Adds, Builds, Creates, Prompts, etc.".

Reply:
Yea, sounds much better